CHANGELOG.md

```diff
@@ -1,3 +1,243 @@
+### citus v11.0.0_beta (March 22, 2022) ###
+
+* Turns MX on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Distributes functions with `CREATE FUNCTION` command
+
+* `citus_shard_indexes_on_worker` shows all local shard indexes regardless of
+  `search_path`
+
+* `citus_shards_on_worker` shows all local shards regardless of `search_path`
+
+* Adds `citus.create_object_propagation` GUC to control ddl creation behaviour
+  in transactions
+
+* Sets `create_object_propagation` default to `immediate`
+
+* Uses a fixed `application_name` while connecting to remote nodes
+
+* Hides shards based on `application_name` prefix
+
+* Prevents `citus.node_conninfo` to use `application_name`
+
+* Starts identifying rebalancer backends by `application_name=citus_rebalancer`
+
+* Starts internal backends by `application_name=citus_internal`
+
+* Adds `citus.enable_unsafe_triggers` flag to propagate triggers on distributed
+  tables
+
+* Adds `citus.max_client_connections` GUC to limit non-Citus connections
+
+* Adds `fix_partition_shard_index_names` udf to fix currently broken names
+
+* Adds propagation for foreign server commands
+
+* Adds propagation of `CREATE AGGREGATE` commands
+
+* Add support for `ALTER FUNCTION ... SUPPORT ...` commands
+
+* Adds support for `CREATE SCHEMA AUTHORIZATION` statements without schema name
+
+* Adds support for `TRUNCATE` for foreign tables
+
+* Adds support for adding partitioned local table to Citus metadata
+
+* Adds support for automatic binary encoding in re-partition joins when possible
+
+* Adds support for foreign tables in MX
+
+* Adds support for operator class parameters in indexes
+
+* Allows `create_distributed_function()` on a function owned by an extension
+
+* Allows creating distributed tables in sequential mode
+
+* Allows disabling nodes when multiple failures happen
+
+* Creates type locally if it has undistributable dependency
```

Comment:
Now we allow doing so for many objects as in https://github.com/citusdata/citus/pull/5836.

I forgot adding a DESCRIPTION line for that pr, so this is my bad. Maybe we can replace this by such a generic one:

Allows locally creating objects having a dependency that we cannot distribute

Reply:
> we cannot distribute

cannot be distributed?

Reply:
I used the following as the 4th item in the changelog:

    * Allows locally creating objects having a dependency that cannot be distributed